src/main/java/io/skai/okta/internshipstreamingserviceparserxml/repository/impl/EpisodeRepository.java

```diff
@@ -0,0 +1,115 @@
+package io.skai.okta.internshipstreamingserviceparserxml.repository.impl;
+
+import io.skai.okta.internshipstreamingserviceparserxml.dto.Episode;
+import io.skai.okta.internshipstreamingserviceparserxml.jooq.generated.tables.records.EpisodesRecord;
+import io.skai.okta.internshipstreamingserviceparserxml.repository.VideoRepository;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.jooq.DSLContext;
+import org.jooq.RecordMapper;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+import java.util.Optional;
+
+import static io.skai.okta.internshipstreamingserviceparserxml.jooq.generated.Tables.EPISODES;
+
+@Repository
+@RequiredArgsConstructor
+@Slf4j
+public class EpisodeRepository implements VideoRepository {
+    private final DSLContext dslContext;
+
+    @Override
+    public void saveOrUpdate(Episode episode) {
+        if (isPresent(episode)) {
+            log.info("update");
+            update(episode);
+        } else {
```

Comment:
instead of if/else statement do return 
`if(sss) {do something; return} do something else`